content/docs/command-reference/status.md

```diff
@@ -157,24 +157,25 @@ that.
 
 ```dvc
 $ dvc status
-bar.dvc:
-        changed deps:
-                modified:      bar
-        changed outs:
-                not in cache:      foo
-foo.dvc:
-        changed outs:
-                deleted:      foo
-        changed checksum
-prepare.dvc:
-        changed outs:
-                new:      bar
-        always changed
+bar:
+	changed deps:
+		modified:           foo
+	changed outs:
+		not in cache:       bar
+foo:
+	changed outs:
+		deleted:            foo
+	changed checksum
+foobar.dvc:
+	changed outs:
+		new:                foobar
+	always changed
 ```
```

Comment:
foobar? Let's go for `baz` (see https://en.wikipedia.org/wiki/Metasyntactic_variable#Common_metasyntactic_variables)

Reply:
p.s. for that matter I'd put foo before bar too.

Reply:
Done